crates/usvg-tree/src/lib.rs

```diff
@@ -737,8 +737,8 @@ impl NodeKind {
     pub fn transform(&self) -> Transform {
         match self {
             NodeKind::Group(ref e) => e.transform,
-            NodeKind::Path(ref e) => e.transform,
-            NodeKind::Image(ref e) => e.transform,
+            NodeKind::Path(_) => Transform::default(),
+            NodeKind::Image(_) => Transform::default(),
             NodeKind::Text(ref e) => e.transform,
```

Comment:
Shouldn't we remove it from text as well?

Reply:
Possibly, kind of forgot about it. Will try.

Reply:
Should we remove the transform method on `Node` all-together now that only groups can have them anyway? Would make a bit more sense, on the other hand it's also convenient if you can just call it on the node itself instead of checking if its a group first.

Reply:
I think we should. Otherwise it's just confusing.